sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -157,4 +157,29 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(Sqrt(-1), null, EmptyRow)
     checkEvaluation(Sqrt(-1.5), null, EmptyRow)
   }
+
+  test("pmod") {
+    checkEvaluation(Pmod(Literal(7), Literal(3)), 1)
+    checkEvaluation(Pmod(Literal(-7), Literal(3)), 2)
+    checkEvaluation(Pmod(Literal(-7), Literal(3)), 2)
+    checkEvaluation(Pmod(Literal(7.0D), Literal(3.0D)), 1.0D)
+    checkEvaluation(Pmod(Literal(7.2), Literal(4.1)), 3.1000000000000005)
```

Comment:
Just want to explicitly declare the specific type. Would change that if it's not the convention.